pymeasure/display/windows.py

```diff
@@ -497,3 +497,396 @@ def finished(self, experiment):
         if not self.manager.experiments.has_next():
             self.abort_button.setEnabled(False)
             self.browser_widget.clear_button.setEnabled(True)
+
+# JM: ############################################################################
+
+# QUESTION: Can we just inherit from ManagedWindow?
```

Comment:
Yes, I think it would be better to inherit from `ManagedWindow` to avoid re-stating code.

Reply:
I think this is worth putting off for another PR.